content/en/kubernetes.md

```diff
@@ -7,49 +7,32 @@ tags: ["infrastructure", "", ""]
 
 ## What it is
 
-Kubernetes, often abbreviated as K8s, is an open source container orchestrator that 
-automates the life-cycle of [containerized](/container/) applications on modern infrastructures. 
-It's like a data center operating system that manages applications 
-running across a [distributed system](/distributed-systems/) 
-(just like the OS on your laptop that manages your apps). 
-
-Kubernetes schedules containers across [nodes](/nodes/) in a [cluster](/cluster/). 
-It bundles several infrastructure constructs, sometimes referred to as “primitives,” 
-like an instance of an app, load balancers, persistent storage, and others together 
-in a way that they can be composed into applications. 
-
-Kubernetes enables automation and extensibility, 
-allowing users to deploy applications declaratively in a reproducible way. 
-Software products and projects in the Kubernetes ecosystem take advantage of that automation and extensibility 
-to extend the Kubernetes [API](/application-programming-interface/). 
-This enables them to leverage Kubernetes’ automation and 
-make their tools more accessible to experienced Kubernetes practitioners.
+Kubernetes, often abbreviated as K8s, is an open source container orchestrator. 
+It automates the lifecycle of containerized applications on modern infrastructures, functioning as a "datacenter operating system" that manages applications across a [distributed system](/distributed-systems/).
+
+Kubernetes schedules [containers](container/) across [nodes](/nodes/) in a [cluster](/cluster/), bundling several infrastructure constructs, sometimes referred to as "primitives." 
+These primitives consist of app instances, load balancers, persistent storage, etc., and are composed into applications.
```

Comment:
The original text says, "composed into an application." What does that mean? Building an app?